app/mailers/order_mailer.rb

```diff
@@ -0,0 +1,11 @@
+class OrderMailer < ApplicationMailer
+
+  def order(customer_email, cart, pickup_time)
```

Comment:
Not sure if I think this method name is suitable. Are there any conventions?

Reply:
None that I found.